content/pages/om-oss.mdx

```diff
@@ -1,10 +1,16 @@
 ---
 title: Om oss
-description: fawekgnaejkgnewkøasg
+description: Queer Hangout er en møteplass for voksne skeive i Drammen. Vi er en frivillig organisasjon som møtes 2 ganger i måneden. Vi jobber mot ensomhet i det skeive miljøet.
 slug: /om-oss
 menu: 1
 ---
 
 # Om oss
 
-Dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+**Queer Hangout** er en møteplass for skeive voksne i Drammen. Vi er en frivillig organisasjon som møtes to ganger i måneden. Vi har også en egen facebookgruppe hvor folk kan legge ut egne aktiviteter, som bursdager, juleverksted, skogsturer eller frokost på 17.mai.  
+  
+Queer Hangout er ikke bare en vennegjeng, men også en gruppe aktivister som jobber mot ensomhet i det skeive miljøet. Blant annet er mange funksjonshemmede utelatt av det skeive miljøet pga. manglende tilrettelegging. Vi har derfor bygget en egen rullestolrampe, ordnet stillerom og valgt tilrettelagte lokaler, sånn at funksjonshemmede kan bli med i Queer Hangout på lik linje som alle andre. Vi samarbeider med hvert medlem og relevante organisasjoner, sånn at vi får gjort ting riktig.  
+
+[Trykk her for å lære mer om våre mål og hvordan **du** kan hjelpe!](https://queerhangout.no/vaare-maal)
+  
+![En frokost på 17.mai, innendørs i en leilighet. 2 personer ser på hverandre og ler, med et glass champagne i hånden. Første personen er i en fin dress, den andre er i samekofte. Det er salat og brødmat over bordet, med 17.mai pynt som små norske flagg](../images/17mai.jpeg)  
```

Comment:
Det bør settes tittel på bildet, omså en tom streng (`""`), slik at ikke alt tekst kommer opp som caption. Det som er vanlig er å sette navnet på personen som har tatt bildet (e.g. `Foto: Leo Imperiatori`)

Jeg forklarer deg ikke hvordan man setter tittel, for jeg så at du fikk det til på bildene av knutepunktet og criollo :)